pyscriptjs/tests/integration/test_py_config.py

```diff
@@ -84,8 +84,8 @@ def test_runtime_config(self, tar_location):
             <py-config type="json">
                 {
                     "runtimes": [{
-                        "src": "/pyodide/pyodide.js",
-                        "name": "pyodide-0.20.0",
```

Comment:
@tedpatrick IMO, we shouldn't remove this test. The reason is that we want to check if passing a runtime from a local path works instead of a hosted one. This is useful when you have the pyodide distribution downloaded and are without an internet access.

Here, we check for `0.20` as another sanity check, to check if indeed load the local and correct one is loaded since the default was `0.21.2`

Reply:
cc: @antocuni

Reply:
@madhur-tandon I need your help finding the zip contents so I can intercept them locally. This is temporary.

Reply:
The tar file (zipped) is stored in a path denoted by the `tar_location` fixture. But the contents of it are extracted to the `self.tmpdir` location inside the `PyScriptTest` class. Hopefully that helps.

Reply:
Fixed